config/src/main/java/com/typesafe/config/impl/AbstractConfigNode.java

```diff
@@ -0,0 +1,15 @@
+package com.typesafe.config.impl;
+
+import com.typesafe.config.ConfigNode;
+
+abstract class AbstractConfigNode implements ConfigNode {
+    final private Token token;
```

Comment:
@havocp As it stands right now, each token lives in its own node. However, some nodes have child nodes instead of Tokens.